.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,9 @@
-**This project uses [semver](http://semver.org), please check the scope of this pr:**
+### This project uses [semver](semver.org), please check the scope of this pr:
```

Comment:
Just a quick look through the repos and I am see more `**`. We can address a consistent approach across the repos in a separate effort.